airflow/models/baseoperator.py

```diff
@@ -1595,11 +1652,14 @@ def chain(*tasks: Union[BaseOperator, Sequence[BaseOperator]]):
 
 
 def cross_downstream(
-    from_tasks: Sequence[BaseOperator], to_tasks: Union[BaseOperator, Sequence[BaseOperator]]
+    from_tasks: Union[Sequence[BaseOperator], Sequence["XComArg"]],
+    to_tasks: Union[Sequence[BaseOperator], Sequence["XComArg"]],
```

Comment:
Shouldn't this be

```suggestion
    from_tasks: Sequence[Union[BaseOperator, "XComArg"]],
    to_tasks: Sequence[Union[BaseOperator Sequence["XComArg"]],
```

Reply:
Actually this:

```suggestion
    from_tasks: Sequence[Union[BaseOperator, "XComArg"]],
    to_tasks: Union[BaseOperator, "XComArg", Sequence[Union[BaseOperator, Sequence["XComArg"]]],
```

Reply:
Good catch, you’re right, these are heterogeneous lists, not two possible homogeneous lists.

Reply:
Agreed, you're right 👍 . Updated.